azkaban-exec-server/src/main/java/azkaban/execapp/ExecutorServlet.java

```diff
@@ -368,6 +371,20 @@ private void setActive(boolean value, Map<String, Object> respMap)
     }
   }
 
+  private void getExecutorId(Map<String, Object> respMap)
+      throws ServletException {
+    try {
+      ExecutorLoader executorLoader = application.getExecutorLoader();
+      Executor executor = executorLoader.fetchExecutor(application.getHost(), application.getPort());
```

Comment:
Also this code is now common for uses, in the Executor server class and in this class, set active method. Should we refactor that into a single method to fetch current executor object ?